hashicorp/boundary-ui
# style: 💄 issues with spacing in manage scopes pages for a role

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-15287

# Description




there was a spacing issue whenever the modal popped up in the UI.

## Screenshots (if appropriate)
before:

https://github.com/user-attachments/assets/828665bb-15f0-4e3d-a9a5-f3cb3b96239b


after:

https://github.com/user-attachments/assets/c9f70bba-f569-4744-8ef9-7686e7b656f3



## How to Test

1. go to a role details view
2. manage -> manage scopes -> manage custom scopes
3. deselect an org that has projects selected to make sure spacing issue with modal is gone.

## Checklist
